LoopKitUI/Views/Settings Editors/SuspendThresholdEditor.swift

```diff
@@ -109,9 +109,7 @@ public struct SuspendThresholdEditor: View {
                                 bounds: viewModel.guardrail.absoluteBounds.lowerBound...viewModel.maxSuspendThresholdValue
                             )
                             // Prevent the picker from expanding the card's width on small devices
-                            .frame(maxWidth: UIScreen.main.bounds.width - 48)
-                            .clipped()
-                            .compositingGroup()
+                            .frame(maxWidth: 220)
```

Comment:
@ps2 This is a bit of an ugly hack but it works around the problem where this picker's unit label was being laid out off screen.